tests/integration/test_db.py

```diff
@@ -365,34 +323,6 @@ async def test_roles_blocking(ops_test: OpsTest, charm: str) -> None:
     )
 
 
-@pytest.mark.group(1)
-@pytest.mark.unstable
-async def test_weebl_db(ops_test: OpsTest, charm: str) -> None:
```

Comment:
Weebl charm doesn't deploy with the legacy charm either.

Reply:
Requested migration officially: https://github.com/canonical/postgresql-operator/pull/405